interpreter/cling/lib/MetaProcessor/MetaSema.cpp

```diff
@@ -151,13 +151,14 @@ namespace cling {
 
       if (expression.empty()) {
         using namespace clang;
+        const char msg[] = "Failed to call `%0%1` to execute the macro.\n"
```

Comment:
```suggestion
        static const char msg[] = "Failed to call `%0%1` to execute the macro.\n"
```

Reply:
Interesting. I tested on godbolt, and the difference is that in the current version, it's packing the string into instructions whereas for the static version it's directly in the object file.
Could you elaborate why you prefer the latter?

Reply:
... because it doesn't go on the stack, probably.